bundles/org.eclipse.cdt.lsp/src/org/eclipse/cdt/lsp/plugin/InitialFileManager.java

```diff
@@ -11,14 +11,12 @@
  * Alexander Fedorov (ArSysOp) - rework to OSGi component
  *******************************************************************************/
 
-package org.eclipse.cdt.lsp.internal;
```

Comment:
component implementation should never be an API, even if we are talking about component scope